[Misc] Add more scoping for improved trace

frank-wei opened the pull request:
Summary: Add more scoping in the hotspot area which can greatly help us to the find the cycle heavy area

Differential Revision: D86436375

<img width="1596" height="414" alt="image" src="https://github.com/user-attachments/assets/dd378dc7-9163-4ded-865b-ae7a2e714dea" />

<img width="2670" height="410" alt="image" src="https://github.com/user-attachments/assets/2384fd03-12de-4c0e-a262-49f0933a0c93" />

bangshengtang commented:
nit: can we try to unify the text format? we have all lower, all upper, first letter capitalized in three different layers

frank-wei committed:
add scoping for better trace (#28329)

Summary:

Add more scoping in the hotspot area which can greatly help us to the find the cycle heavy area

Reviewed By: henryoier

Differential Revision: D86436375

Signed-off-by: Wei Wei <wwei6@meta.com>

frank-wei committed:
pre-commit

Signed-off-by: Wei Wei <wwei6@meta.com>

frank-wei committed:
precommit

Signed-off-by: Wei Wei <wwei6@meta.com>

frank-wei committed:
Merge branch 'main' into export-D86436375

Signed-off-by: Wei Wei <wwei6@meta.com>
